front/src/applications/operationalStudies/hooks/useAutoUpdateProjection.ts

```diff
@@ -23,17 +23,19 @@ import { formatEditoastTrainIdToTrainScheduleId, isTrainSchedule } from 'utils/t
  */
 const useAutoUpdateProjection = (
   infra: InfraWithState,
-  trainIds: number[],
+  editoastTrainIds: number[],
   trainScheduleSummaries: TrainScheduleWithDetails[]
 ) => {
   const dispatch = useAppDispatch();
   const currentTrainIdForProjection = useSelector(getTrainIdUsedForProjection);
   const selectedTrainId = useSelector(getSelectedTrainId);
 
-  const formattedTrainIds = trainIds.map((id) => formatEditoastTrainIdToTrainScheduleId(id));
+  const formattedTrainIds = editoastTrainIds.map((editoastTrainId) =>
```

Comment:
nit (back to the original name of the const)
```suggestion
  const trainIds = editoastTrainIds.map((editoastTrainId) =>
```

Reply:
Hmm I think keeping the term `formatted` would easier alert someone, who isn't familiar with the paced train issues, that we are manipulating a different kind of id.